### PR TITLE
Remove normalization in Pytorch Image Segmentation example

### PR DESCRIPTION
**Pull request by yeandy**
Fixes https://github.com/apache/beam/issues/22224. I mistakenly added a normalization operation during the preprocessing step. This has been removed. The actuals [file](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/ml/inference/pytorch_inference_it_test.py#L117) has also been overwritten with the updated outputs.

This should also fix the issue filed here https://github.com/apache/beam/issues/22413 because of a premature update of the actuals file.

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] [**Choose reviewer(s)**](https://beam.apache.org/contribute/#make-your-change) and mention them in a comment (`R: @username`).
 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://beam.apache.org/contribute/#make-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI.

### CODE REVIEW

**Comment by yeandy** [comment]
Run Python 3.7 PostCommit

**Comment by yeandy** [comment]
Run Python 3.9 PostCommit

**Comment by yeandy** [comment]
remind me after tests pass

**Comment by yeandy** [comment]
R: @AnandInguva

**Comment by yeandy** [comment]
Run Python 3.7 PostCommit

**Comment by yeandy** [comment]
Run Python 3.9 PostCommit

**Review by AnandInguva — APPROVED** [review]

File: sdks/python/apache_beam/examples/inference/pytorch_image_segmentation.py [hunk 1]
```diff
@@ -147,12 +147,9 @@ def preprocess_image(data: Image.Image) -> torch.Tensor:
   image_size = (224, 224)
   # Pre-trained PyTorch models expect input images normalized with the
```
[the remark this answers is not in the document]
Comment (yeandy) [reply sdks/python/apache_beam/examples/inference/pytorch_image_segmentation.py]: Done.

**Comment by yeandy** [comment]
Run Python 3.7 PostCommit

**Comment by yeandy** [comment]
Run Python 3.9 PostCommit

**Comment by yeandy** [comment]
Failures are unrelate. Local gradle test passes.

**Comment by yeandy** [comment]
R: @TheNeuralBit

**Review by TheNeuralBit — APPROVED** [review]
LGTM, but I'd like to get a green run on PostCommits before merging

**Comment by AnandInguva** [comment]
One of the test is fixed recently. Let's rerun the post commits again

**Comment by AnandInguva** [comment]
Run Python 3.7 PostCommit

**Comment by AnandInguva** [comment]
Run Python 3.9 PostCommit

**Comment by yeandy** [comment]
Run Python 3.7 PostCommit

**Comment by yeandy** [comment]
Run Python 3.9 PostCommit

**Comment by yeandy** [comment]
Tests pass @TheNeuralBit @tvalentyn

**Comment by TheNeuralBit** [comment]
Thanks @yeandy!

**Pull request merged**